app/assets/stylesheets/components/_card.scss

```diff
@@ -55,8 +55,7 @@
 }
 
 .card-border {
-  border-style: solid;
-  box-shadow: 0 0 15px rgba(0,0,0,0.2);
+  border: solid;
```

Comment:
i got rid of the box shadow - personally not a fan of it but if you want can keep it